lib/help.js

```diff
@@ -36,6 +36,7 @@ Evergreen update:
   evergreen <project_path>      => generate evergreen update ZIPs
                                      omit <project_path> to use working dir
     --force                     => overwrite any existing update ZIPs
+                                     omit to be prompted for confirmation
```

Comment:
Shorter to state `-- force => overwrite existing files without confirmation` ? Just assume that the default behaviour is the opposite (with confirmation)?